### PR TITLE
Feature Main and About page

### PR DESCRIPTION
**Pull request by gitbygaurav**
### **In this branch I :**

- I have used tailwind css
- Added home page,
- Added about page,
- Created responsive styles for min: 768px screen sizes,
- Add dynamic featured artists section with a javasript object whicah contains list' data.
- Deploy the page using Github pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,128 @@
...
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here is a link to the live demo.
+
+- [Live Demo Link]()
+
+> Here is a link to the project details video.
+
+- [Loom Video Link](https://www.loom.com/share/a30c9bb4d5ac4d6492005886b4ec3152)
```
Comment (OLIPLICHE) [inline README.md]: - Please record again, make sure the video contains a highlight about a piece of the code you find interesting or proud of making.

File: README.md [hunk 2]
```diff
@@ -1 +1,128 @@
...
+
+- Responsive 
+- Fast
+- Modern design
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (OLIPLICHE) [inline README.md:49]: - Please makes use of GitHub Pages to deploy the web pages. The project must be deployed properly, and all pages, links, and images work as expected.
- Provide the link to the deployed project here in ReadMe.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      <img class="h-[15vh] md:h-[24vh]" src="./Assets/4117741.svg" alt="logo">
+      <div>
+        <p class="text-white">Copyright ©️ 2022 Lorem ipsum Travels. Some Rights Reserved</p>
+        <p class="md:flex hidden text-white text-center w-80">Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+          A, totam minima tempore ipsam est sint modi, voluptatum suscipit praesentium vitae inventore temporibus neque
+          doloremque deleniti reiciendis assumenda blanditiis. Odit, facilis!</p>
+      </div>
+    </div>
+  </footer>
+  <script src="main.js"></script>
```
Comment (OLIPLICHE) [inline about.html]: # Mobile Size: About.
- Please make sure mobile size has a white color.
- Kindly add Partners section.
- See the Original Design:

Capture1
- Look at your design:

Capture

File: style.css [hunk 4]
```diff
@@ -0,0 +1,175 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap');
+
```
Comment (OLIPLICHE) [inline style.css:4]: # Mobile Size: Abou
- Please try to keep each section inside the block and not overtake another block. 
- **See your Job**:
- 
Capture
-
- 
**Have a look at the Original design:**
Capture1

File: index.html [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+        </div>
+        <div class="flex gap-5 py-2">
+            <p class="text-white">English</p>
+            <p class="text-white">My Page</p>
+            <p class="text-white">Logout</p>
+        </div>
+    </div>
+    <nav class="hidden bg-white md:flex justify-around shadow-xl">
+        <div>
+            <img class="h-20" src="./Assets/4117741.svg">
```
Comment (OLIPLICHE) [inline index.html]: - Please make sure the image contains `alt` attributes

File: about.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+      </div>
+      <div class="flex gap-5 py-2">
+        <p class="text-white">English</p>
+        <p class="text-white">My Page</p>
+        <p class="text-white">Logout</p>
+      </div>
+    </div>
+    <nav class="hidden bg-white md:flex justify-around shadow-xl">
+      <div>
+        <a href="./index.html"><img class="h-20" src="./Assets/4117741.svg"></a>
```
Comment (OLIPLICHE) [inline about.html]: - Please make sure the image contains `alt` attributes

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @gauravgangwarweb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️
- Please I have provided the link to [the Original Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). It may help
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @gauravgangwarweb   

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request ✔️ 
- design well followed✔️ 
- Readme up to task✔️ 
- Great project idea✔️ 
- No linter errors✔️ 

<be>
-- Always try to keep your video presentation simple and straight to the point so you touch all project parts in the limited time. nice work

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 3 above)
Comment (gitbygaurav) [reply about.html]: Thanks for the suggestions I have fixed the issue

File: style.css (hunk 4 above)
Comment (gitbygaurav) [reply style.css:4]: Thanks for the suggestions I have fixed the issue

File: index.html (hunk 5 above)
Comment (gitbygaurav) [reply index.html]: Thanks for the suggestions I have fixed the issue

File: about.html (hunk 6 above)
Comment (gitbygaurav) [reply about.html]: Thanks for the suggestions I have fixed the issue

**Comment by gitbygaurav** [comment]
Hii @tufoinnkuo10 👋

Thanks for reviewing and approving my PR

**Pull request merged**